README.md

```diff
@@ -9,26 +9,25 @@ Read more [here](http://datafordemocracy.org/projects/ethics.html).
 ## Ethics Principles
 ---
 <ul>
```

Comment:
A high level point of feedback: This document should take on either a "pledge" or a "directive" format, across all principles. Example of a "pledge" format: the hippocratic oath (https://en.wikipedia.org/wiki/Hippocratic_Oath). Example of a "directive" format: the zen of python (https://en.wikipedia.org/wiki/Zen_of_Python).

In this PR I edited them to the "pledge" format for the most part, but it is just a starting point.

Reply:
I love that you have taken advantage of the pr flow/commenting system to open a dialogue about this project. Thank you!